components/user/UserProfile.js

```diff
@@ -73,7 +77,8 @@ function UserProfile({ BASE_URL, data }) {
         </div>
       )}
 
-      <div className="flex justify-center my-4">
+      {/* Passed Ref object as the ref attribute to the JSX of the DOM node of QR */}
+      <div className="flex justify-center my-4" ref={qrRef}>
         {qrShow && (
           <QRCodeCanvas
```

Comment:
The id can be removed form this component as it is no longer used

Reply:
Yes Done removed